src/NuGet.Clients/NuGet.CommandLine/Common/Verbosity.cs

```diff
@@ -1,10 +1,3 @@
-﻿
-namespace NuGet.CommandLine
-{
-    public enum Verbosity
-    {
-        Normal,
-        Quiet,
-        Detailed
-    }
-}
+using System.Runtime.CompilerServices;
+
+[assembly:TypeForwardedTo(typeof(NuGet.CommandLine.Verbosity))]
```

Comment:
I wouldn't imagine this needs forwarded. 

The core components have a LogLevel, this is a client implementation detail. For example dotnet.exe has more log levels than this, even if we don't respect them.

Reply:
You are recommending I move over to use another existing type, like LogLevel. Let me look.
Related to the compat or not/library question as well.

Reply:
fixing. not loving it, but fixing it.